library/src/test/java/io/constructor/core/ConstructorIoIntegrationTest.kt

```diff
@@ -63,10 +67,43 @@ class ConstructorIoIntegrationTest {
         Thread.sleep(timeBetweenTests)
     }
 
+    @Test
+    fun getAutocompleteResultsCRTAgainstRealResponse() {
+        runBlocking {
+            launch {
```

Comment:
Is there anything gained from using both `runBlocking` and `launch`?
It seems to me that this would work the same without the `launch` block.
I'd say `launch` would only be useful if there were multiple statements (multiple different `launch`es).

Reply:
I'm not exactly sure, but I just tested it without `launch` and it seems to work. I know that `runBlocking` blocks the current thread until the code (or suspended fns) inside it runs to completion. It seems like both they both start a new coroutine scope, so `launch` doesn't seem necessary here. I'll go ahead and remove it. Thanks!

Reply:
Yeah, `runBlocking` will block, but since we're not really doing anything in parallel it's not necessary to have `launch` within it :+1: